File: ApiResources/src/NfcCardAssignments/domain/entities/NfcCardAssignment.go
package entities

import "time"

type NfcCardAssignment struct {
    ID         int       `json:"id"`
    UserID     int       `json:"user_id"`
    CardUID    string    `json:"card_uid"`
    AssignedAt time.Time `json:"assigned_at"`
    IsActive   bool      `json:"is_active"`
}

File: Access-Gate-APP/Access-Gate/src/app/features/admin/models/NFCCardAssignment.ts
import { User } from './IUsers';
import { NfcCard } from './iNfc_cards';

export interface NFCCardAssignment {
  id: number;
  user_id: User['id'];
  card_uid: NfcCard['card_uid'];
  assigned_at: Date;
  is_active: boolean;
}